modules/backend/widgets/Calendar.php

```
@@ -0,0 +1,1042 @@
+     * It has been called from the first refresh page
+     * and next or previous button click event
+     *
+     * @return json
+     */
+    public function onRefreshEvents()
+    {
+        $startTime = post('startTime');
+        $endTime = post('endTime');
+        $timeZone = post('timeZone');
+
+        $data = [
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)